src/h5web/toolbar/controls/DomainSlider/DomainTooltip.tsx

```diff
@@ -65,66 +74,74 @@ const DomainTooltip = forwardRef<Handle, Props>((props, ref) => {
       hidden={!open}
     >
       <div className={styles.tooltipInner}>
-        {minGreater && (
-          <ErrorMessage
-            error={DomainError.MinGreater}
-            showSwapBtn={!isAutoMin && !isAutoMax}
-            onSwap={onSwap}
+        {histogram && (
+          <DataHistogram
+            bins={histogram.bins}
+            data={histogram.data}
+            size={controlsSize}
           />
         )}
+        <div ref={controlsRef} className={styles.tooltipControls}>
```

Comment:
Crappy diff here: I put all the controls into this `div` to be able to position the histogram on the left of the controls